content/en/blog/2023/opentelemetry-in-focus-january-2023.md

```diff
@@ -0,0 +1,92 @@
+---
+title: OpenTelemetry in Focus - January 2023
+linkTitle: OpenTelemetry in Focus - January 2023
+date: 2023-01-31
+author: >-
+  [Austin Parker](https://github.com/austinlparker)
+---
+
+Welcome to the first edition of OpenTelemetry In Focus! This blog is intended to
+be an overview of important releases, roadmap updates, and community news. This
+series will be focusing on our core components, such as the specification, data
+format, tools, and most popular API/SDKs.
+
+Are you a maintainer with something you’d like featured here? Please get in
+touch with me via email, or on the CNCF Slack!
```

Comment:
In addition to the suggestion given below, consider adding the link to the CNCF Slack.

```suggestion
Are you a maintainer with something you’d like featured here? Get in
touch with me via email, or on the CNCF Slack!
```

How are folks meant to get your email address?